### PR TITLE
Fixes #695: Fix Extra Spacing.

### LINKED ISSUE #688: Lowfi: Update topic_lessons_story_summary
Mock: https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/c718fc6d-b336-475a-b31b-fd3bc84882a8/PM-EN-Play-Tab-Progress-1-

In new mock the `topic_lessons_story_summary` has been updated to remove the extra space before chapter names.

Current Develop implementation:
Screenshot 2020-02-22 at 5 49 27 PM

New updated implementation:
Screenshot 2020-02-22 at 5 49 33 PM

So if you see, the space before the chapter names is different in both the mocks.

### PR DESCRIPTION
**Pull request by gunishjain**
## Explanation
Fixes #695: Fix Extra Spacing.
Fixes extra space between chapter names. 
Simply changed minHeight value to 20dp

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by gunishjain** [comment]
@rt4914 PTAL

**Comment by PrarabdhGarg** [comment]
@gunishjain please update the PR title to Fixes #695: Fix Extra Spacing.

**Comment by gunishjain** [comment]
@PrarabdhGarg Done

**Comment by PrarabdhGarg** [comment]
@gunishjain Not under the Explanation section. Rather the title that appears in bold at the top of your screen.
Capture
This one

**Comment by gunishjain** [comment]
Oops. Sorry this is my first time contributing @PrarabdhGarg. Please check now

**Comment by PrarabdhGarg** [comment]
Also, please revert the changes made to .idea/misc.xml and .idea/codeStyles/Project.xml. These files might have been automatically changed by your android studio, but these changes should not be merged in the develop branch.

**Comment by gunishjain** [comment]
@PrarabdhGarg  I don't know how to revert so I just deleted those files from the commit. Next time I will not include these types of changes.

**Comment by rt4914** [comment]
> @gunishjain please update the PR title to Fixes #695: Fix Extra Spacing.

@PrarabdhGarg @gunishjain Actually the PR title should be `Fix #688: Update extra spacing in topic_lessons_story_summary`. 
Also, the issue that you are fixing is 688 and not 695(which is this same PR by the way)

We write `Fix` when we are solving an entire issue and we write `Fixes` when are solving just a single part of an issue and not the entire issue.

**Comment by rt4914** [comment]
> @PrarabdhGarg I don't know how to revert so I just deleted those files from the commit. Next time I will not include these types of changes.

Actually we need those files, so therefore you cannot delete those files. Now you can fix this by adding those files back to the code taking it from develop branch. Also, once you do that it might happen that there will be changes in these files automatically because of android studio. In that case, instead of using `git add .` (which basically adds all changed files), you can just add the files that you want to add using `git add <here comes file name along with path>`

File: app/src/main/java/org/oppia/app/topic/lessons/ChapterSummaryAdapter.kt [hunk 1]
```diff
@@ -43,4 +43,6 @@ class ChapterSummaryAdapter(
       }
     }
   }
+
+
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/topic/lessons/ChapterSummaryAdapter.kt]: Remove these two extra blank lines, we don't need these lines.

File: app/src/main/res/layout/lessons_chapter_view.xml [hunk 2]
```diff
@@ -21,7 +21,7 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:clickable="@{chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? true: false}"
-    android:minHeight="48dp"
+    android:minHeight="20dp"
```
Comment (rt4914) [inline app/src/main/res/layout/lessons_chapter_view.xml:24]: This is incorrect. We want this to be exactly same as before. 
As the issue describes you have to make change to `topic_lessons_story_summary` not this file. Also, we want the starting margin to be removed not the top/bottom.
In the above mentioned files you mainly need to make some change to `RecyclerView` to fix the issue.

**Review by rt4914 — CHANGES_REQUESTED** [review]
PTAL. Thanks @gunishjain 

Also, thanks @PrarabdhGarg for helping us out with this review. We appreciate this.

File: app/src/main/res/layout/lessons_chapter_view.xml (hunk 2 above)
Comment (gunishjain) [reply app/src/main/res/layout/lessons_chapter_view.xml:24]: Actually I was not able to find the code where recycler view is initilized you must be talking about adding Item decorator to the recycler view so I was thinking to make another fresh PR. Please remove this one or should I do it from my end?

Comment (rt4914) [reply app/src/main/res/layout/lessons_chapter_view.xml:24]: > Actually I was not able to find the code where recycler view is initilized you must be talking about adding Item decorator to the recycler view so I was thinking to make another fresh PR. Please remove this one or should I do it from my end?

I am not talking about the item decorator, you just need to change the margin inside RecyclerView to make the left side margin(extra space) disappear. 

Also, you can close this PR, but clicking on `Close Pull Req`

**Pull request closed without merging**